doc/mgr/dashboard.rst

```diff
@@ -1,43 +1,36 @@
-Dashboard Plugin
-================
+Ceph Manager Dashboard
+======================
 
 .. _mgr-dashboard-overview:
 
 Overview
 --------
 
-The original Ceph manager dashboard that was shipped with Ceph "Luminous"
+The original Ceph Manager Dashboard that was shipped with Ceph Luminous
 started out as a simple read-only view into various run-time information and
 performance data of a Ceph cluster. It used a very simple architecture to
-achieve the original goal.
-
-However, there was a growing demand for adding more web-based management
-capabilities, to make it easier to administer Ceph for users that prefer a WebUI
-over using the command line.
-
-This new dashboard module is a replacement of the previous one and an ongoing
-project to add a native web based monitoring and administration application to
-Ceph Manager.
-
-The architecture and functionality of this module are derived from and inspired
-by the `openATTIC Ceph management and monitoring tool
-<https://openattic.org/>`_. The development is actively driven by the team
-behind openATTIC at SUSE.
-
-The intention is to reuse as much of the existing openATTIC functionality as
-possible, while adapting it to the different environment. While openATTIC is
-based on Django and the Django REST Framework, the dashboard module's backend
-code uses the CherryPy framework and a custom REST API implementation.
-
-The WebUI implementation is based on Angular/TypeScript, merging both
-functionality from the original dashboard as well as adding new functionality
-originally developed for the standalone version of openATTIC.
-
-The dashboard plugin is implemented as a web application that visualizes
-information and statistics about the Ceph cluster using a web server hosted by
-``ceph-mgr``.
-
-The dashboard currently provides the following features to monitor and manage
+achieve the original goal. However, there was a growing demand for adding more
+web-based management capabilities, to make it easier to administer Ceph for users
+that prefer a WebUI over using the command line.
+
+The new :term:`Ceph Manager Dashboard` plugin is a replacement of the previous
+one and adds a native web based monitoring and administration application to
+the Ceph Manager. The architecture and functionality of this new Ceph Manager
+Dashboard plugin is derived from and inspired by the
+`openATTIC Ceph management and monitoring tool
+<https://openattic.org/>`_. The development is actively driven
+by the team behind openATTIC at `SUSE <https://www.suse.com/>`_. The intention
+is to reuse as much of the existing openATTIC functionality as possible, while
+adapting it to the different environment. While openATTIC is based on Django and
```

Comment:
I'm wondering if it's possible to replace the second ```while```. Maybe by ```Although openATTIC is based on Django and ...```?

Reply:
Suggestion: "openATTIC is based on Django and the Django REST Framework, _whereas_ the new Dashboard Plugin's backend code uses [..]"